webapp/app.py

```diff
@@ -1,29 +1,50 @@
+from webapp.tasks import LOCKS
+from webapp.sso import login_required
+from webapp.site_repository import SiteRepository
+from webapp.schemas import (
+    ChangesRequestModel,
+)
+from webapp.models import (
+    Reviewer,
+    Webpage,
+    JiraTask,
+    db,
+    get_or_create,
+    WebpageStatus
+)
```

Comment:
these imports should be lower, but its likely that many more older files have unsorted imports - that's probably the job of a lint checker.

Reply:
I'll create an issue to reformat the python files, and add sorted import checks to the CI